src/components/EditableTypography/EditableTypography.tsx

```diff
@@ -109,22 +114,27 @@ const EditableTypography: VibeComponent<EditableTypographyProps, HTMLElement> =
       }
     }, [isEditing]);
 
+    const AnimatedTypography = motion(TypographyComponent);
+    const isOverflowing = useIsOverflowing({ ref: typographyRef, ignoreHeightOverflow: true });
+    const shouldAnimate = useMemo(() => isOverflowing && isEditing !== null, [isEditing, isOverflowing]);
+
     return (
-      <div
-        ref={mergedRef}
-        id={id}
-        aria-label={ariaLabel}
-        data-testid={dataTestId}
-        className={cx(styles.editableTypography, className)}
-      >
-        {isEditing && !readOnly ? (
-          <>
-            <HiddenInputPlaceholder
-              className={cx(styles.input, typographyClassName)}
-              value={inputValue || placeholder}
-              onChange={setInputWidth}
-            />
-            <input
+      <LazyMotion features={domAnimation}>
```

Comment:
Motion is lazy loaded (and it's tree shakable, plus we take only the `m` function and not the `motion` function which is the light version of it, see [this](https://www.framer.com/motion/guide-reduce-bundle-size/#how-to-reduce-the-size-of-the-motion-component) for more info

Reply:
If it's not loaded the animation just isn't shown